NETCORE/test/MVCFramework20.FunctionalTests/MVCFramework20.FunctionalTests20.csproj

```diff
@@ -6,7 +6,7 @@
 	<TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
 	<RuntimeIdentifier Condition=" '$(TargetFramework)' == 'net461' ">win7-x86</RuntimeIdentifier>
     <PreserveCompilationContext>true</PreserveCompilationContext>
-    <AssemblyName>MVCFramework20.FunctionalTests20</AssemblyName>
+    <AssemblyName>MVCFramework20.FunctionalTests20.Tests</AssemblyName>
```

Comment:
https://github.com/microsoft/ApplicationInsights-dotnet/blob/develop/NETCORE/test/MVCFramework20.FunctionalTests/FunctionalTest/CorrelationMvcTests.cs#L14

The name of the assembly is hardcoded in the above line. So changing it should have broken the tests. If it didn't I am  confused.... This needs to be addressed.

Reply:
it looks like a bunch of the tests did fail.
Thanks for sharing, i'll look into this